asserts/store_asserts.go

```diff
@@ -0,0 +1,94 @@
+package asserts
+
+import (
+	"fmt"
+	"net/url"
+)
+
+// EnterpriseStore holds an enterprise-store assertion, defining the
+// configuration needed to connect a device to the enterprise store.
+type EnterpriseStore struct {
+	assertionBase
+	address *url.URL
+}
+
+// OperatorID returns the account id of the enterprise store's operator.
+func (estore *EnterpriseStore) OperatorID() string {
+	return estore.HeaderString("operator-id")
+}
+
+// Store returns the identifying name of the operator's enterprise store.
+func (estore *EnterpriseStore) Store() string {
+	return estore.HeaderString("store")
```

Comment:
So, I think he had 2 main concerns:

1. Using something a bit too name-like in the PK. It's really just an identifier, and will typically be "default" as we previously discussed and agreed.
2. Confusion with the model's `store`.

(2) is the bigger problem I think, as you just highlighted - they're totally different things. The hope is that all devices will have a `model` one day; the model's `store` will dictate the store tenant the device gets snaps from. A custom-store's `store` is purely an identifier to allow an operator to differentiate between stores if they need 2+; it does not override a device's store tenant.